packages/embed_api/src/createlyEmbed.ts

```diff
@@ -0,0 +1,187 @@
+import {PostMessageAPI} from './message-api';
+import { of, Observable, BehaviorSubject } from 'rxjs';
+import { switchMap } from 'rxjs/operators';
+import { throwError } from 'rxjs';
+
+interface Doctype {
+    iFrameId:string;
```

Comment:
Should be container element id,
The iframe should be appended to the element specified by this id